apps/web/src/services/cost-explorer/components/CostAnalysisChart.vue

```diff
@@ -53,6 +53,7 @@ const state = reactive({
     showHideAll: computed(() => Object.values(state.legend).some((d) => d)),
     showAccumulatedToggle: computed(() => costAnalysisPageState.granularity === GRANULARITY.DAILY),
     isAccumulated: false,
+    analyzeFetcher: computed(() => (costAnalysisPageGetters.isUnifiedCost ? unifiedCostAnalyze : fetchCostAnalyze)),
```

Comment:
Question: Is it a good approach to manage the fetcher as a state?

Reply:
Answered verbally.

Q: Is it okay to include a function that is not a pure value in the state?
A: This is an unrelated case because it returns a fixed function with no content that can be changed as a state in the internal logic of the function.